homeassistant/components/maxcube/binary_sensor.py

```diff
@@ -11,13 +11,10 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     """Iterate through all MAX! Devices and add window shutters."""
     devices = []
     for handler in hass.data[DATA_KEY].values():
-        cube = handler.cube
-        for device in cube.devices:
-            name = f"{cube.room_by_id(device.room_id).name} {device.name}"
-
+        for device in handler.cube.devices:
```

Comment:
Can we move the changes to the integration to a separate PR and just do the tests in this PR? It's better to add the tests first so we see that the tests still pass when making changes in the integration.

Reply:
Done